.github/workflows/lint.yaml

```
@@ -0,0 +1,21 @@
+name: Lint
+on: [push, pull_request]
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
```

It might be good to add a cache after checkout:
```suggestion

    - name: Cache
      uses: actions/cache@v2
      with:
        path: ~/.npm
        key: ${{ runner.os }}-npm-lock-${{ hashFiles('**/package-lock.json') }}
        restore-keys: |
          ${{ runner.os }}-npm-lock-

```

---

Looks like the setup-node can do this under the hood: https://github.com/actions/setup-node#caching-packages-dependencies, I'll turn it on